Library/Homebrew/hardware.rb

```diff
@@ -55,6 +55,8 @@ def self.cores_as_words
     when 1 then "single"
     when 2 then "dual"
     when 4 then "quad"
+    when 6 then "hexa"
```

Comment:
"Single" and "dual" are not Latin/Greek prefixes, and also they're already not of consistent style.  (That would require pairing "single" with "double".)  Even if we ignore that, the closest stylistic extension would be "quadruple", "sextuple", "octuple".

We're just not gonna get sylistically consistent prefixes.

In English, "hexacore" is about a thousand times more awkward then "hexcore" or "hex core" or "hex-core", and those are about a thousand times more awkward than "six core" and "six-core".